src/controllers/user.js

```diff
@@ -0,0 +1,142 @@
+import Users from '../models/User.js';
+import { isValidObjectId } from 'mongoose';
+
+export const getUsers = async (req, res) => {
+    try{
+        const users = await Users.find(req.query);
+        if(!users) {
+            throw new Error({
+                message: 'Users not found',
+                status: 404,
```

Comment:
change this throw error for:

`return res.status(404).json({
message: 'Users not found',
error: true,
data: undefined
})
`